srtcore/core.h

```diff
@@ -385,7 +388,8 @@ class CUDT
     int64_t maxBandwidth() const { return m_llMaxBW; }
     int MSS() const { return m_iMSS; }
 
-    uint32_t latency_us() const {return m_iTsbPdDelay_ms*1000; }
+    uint32_t peer_latency_us() const {return m_iPeerTsbPdDelay_ms * 1000; }
+    int peer_idle_tout_ms() const { return m_iOPT_PeerIdleTimeout; }
```

Comment:
This, and `peer_latency_us`: naming convention dictates using `peerLatency_us` and `peerIdleTout_ms` (i'd also rather use `peerIdleTmout_ms`).